### ISSUE TITLE
RHEL converge fails due to 'wpars' library not existing

### ISSUE THREAD

**Issue by johnbellone** [chef-cookbooks/aix#38]
### Cookbook version
1.0.0

### Chef-client version
12.9.38

### Platform Details
RHEL 6.6

### Scenario:
It seems that [this change](https://github.com/chef-cookbooks/aix/commit/515b9f6c243f88427dc5a6b0be736f93069c993a) has added the 'wpars' as a requirement that isn't available. This is probably something that needs to happen in a recipe or at least lazily done in the provider when it is explicitly added to resource collection. 

### Steps to Reproduce:
Just include this cookbook on a vanilla CentOS run-list. 

### Expected Result:
No-op

### Actual Result:
Boom goes the dynamite. 

```
       Installing Cookbook Gems:
       Compiling Cookbooks...

       ================================================================================
       Recipe Compile Error in /tmp/kitchen/cache/cookbooks/aix/providers/wpar.rb
       ================================================================================

       LoadError
       ---------
       cannot load such file -- wpars

       Cookbook Trace:
       ---------------
         /tmp/kitchen/cache/cookbooks/aix/providers/wpar.rb:20:in `class_from_file'

       Relevant File Content:
       ----------------------
       /tmp/kitchen/cache/cookbooks/aix/providers/wpar.rb:

        13:  #
        14:  # Unless required by applicable law or agreed to in writing, software
        15:  # distributed under the License is distributed on an "AS IS" BASIS,
        16:  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
        17:  # See the License for the specific language governing permissions and
        18:  # limitations under the License.
        19:  require 'chef/mixin/shell_out'
        20>> require 'wpars'
        21:
        22:  include Chef::Mixin::ShellOut
        23:
        24:  use_inline_resources
        25:
        26:  # support whyrun
        27:  def whyrun_supported?
        28:    true
        29:  end

       Platform:
       ---------
       x86_64-linux


       Running handlers:
       [2016-07-15T13:03:01+00:00] ERROR: Running exception handlers
       Running handlers complete
       [2016-07-15T13:03:01+00:00] ERROR: Exception handlers complete
       Chef Client failed. 0 resources updated in 04 seconds
       [2016-07-15T13:03:01+00:00] FATAL: Stacktrace dumped to /tmp/kitchen/cache/chef-stacktrace.out
       [2016-07-15T13:03:01+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
       [2016-07-15T13:03:01+00:00] ERROR: cannot load such file -- wpars
       [2016-07-15T13:03:05+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by johnbellone** [comment]
/cc @adejoux

This error creeped in when the [WPAR custom resource](https://github.com/chef-cookbooks/aix/pull/35) was added. Is this a gem that needs to be added that we can require?

**Comment by adejoux** [comment]
Hello,

Yes, you are correct :) . I added this gem when I added the wpar resource.
I didn't find a way to make this requirement conditional.

it's included in the current gemfile of this cookbook:
 gem 'aix-wpar', '>=0.1.0'

https://github.com/chef-cookbooks/aix/blob/master/Gemfile#L23

**Comment by johnbellone** [comment]
This is now broken on Supermarket with the latest release.

**Comment by jkohlmeier** [comment]
is there a fix for this yet?

**Comment by tas50** [comment]
This is fixed in the 1.2 release
